### PR TITLE
Outdated single client information at chrome.debugger

### LINKED ISSUE #3901: Single client note in chrome.debugger documentation no longer relevant
https://developer.chrome.com/docs/extensions/reference/debugger/

The note says:
> As of today, attaching to the tab by means of the debugger API and using embedded Chrome DevTools with that tab are mutually exclusive. If user invokes Chrome DevTools while extension is attached to the tab, debugging session is terminated. Extension can re-establish it later.

From Chrome 63 update notes (https://developer.chrome.com/blog/new-in-devtools-63/#multi-client):
> Chrome Extensions using the chrome.debugger API can now run at the same time as DevTools.

So it seems like the note in chrome.debugger documentation was only relevant before v63.

### PR DESCRIPTION
**Pull request by AaronForinton**
Fixes #3901

Source:

From Chrome 63 update notes (https://developer.chrome.com/blog/new-in-devtools-63/#multi-client):

> Chrome Extensions using the chrome.debugger API can now run at the same time as DevTools.

> As of Chrome 63, DevTools now supports multiple remote debugging clients by default, no configuration needed.

### CODE REVIEW

**Comment by jpmedley** [comment]
@AaronForinton, did you test this?

File: site/en/docs/extensions/reference/debugger/index.md [hunk 1]
```diff
@@ -3,11 +3,7 @@ api: debugger
 has_warning: This permission <a href="/docs/extensions/mv3/permission_warnings/#permissions_with_warnings">triggers a warning</a>.
 ---
 
-## Notes
-
-As of today, attaching to the tab by means of the debugger API and using embedded Chrome DevTools
-with that tab are mutually exclusive. If user invokes Chrome DevTools while extension is attached to
-the tab, debugging session is terminated. Extension can re-establish it later.
+## Note
```
Comment (jpmedley) [inline site/en/docs/extensions/reference/debugger/index.md:6]: @dotproto, it seems like we should just remove 'as of today' and start the note at 'Attaching'. What do you think?

**Comment by AaronForinton** [comment]
@jpmedley, I haven't used the API but just wanted to get a conversation going based on #1994 and #3901

File: site/en/docs/extensions/reference/debugger/index.md (hunk 1 above)
Comment (jpmedley) [reply site/en/docs/extensions/reference/debugger/index.md:6]: Ping!

Comment (dotproto) [reply site/en/docs/extensions/reference/debugger/index.md:6]: As I recall, Chrome now supports having multiple debuggers attached simultaneously, but I don't know when multi-debugger support landed. As such, I think it's best to remove the lines as originally suggested in this PR.

Comment (AaronForinton) [reply site/en/docs/extensions/reference/debugger/index.md:6]: @dotproto Is it [this update](https://developer.chrome.com/blog/new-in-devtools-63/#multi-client) from M63?

**Review by jpmedley — APPROVED** [review]

**Pull request merged**